pkg/utils/experimentHelpers.go

```diff
@@ -84,11 +112,11 @@ func (expDetails *ExperimentDetails) ValidateSecrets(clients ClientSets) error {
 			log.Infof("Incomplete Information in Secret, will abort execution")
 			return errors.New("Abort Execution")
 		}
-		err := clients.ValidateConfigMap(v.Name, expDetails)
+		err := clients.ValidateSecrets(v.Name, expDetails)
 		if err != nil {
-			log.Infof("Unable to get ConfigMap with Name: %v, in namespace: %v", v.Name, expDetails.Namespace)
+			log.Infof("Unable to get Secrets with Name: %v, in namespace: %v", v.Name, expDetails.Namespace)
```

Comment:
Unable to list secret "x" in namespace "y", skipping execution

Reply:
Done.